### PR TITLE
Append CoC code to each project in filter

### LINKED ISSUE #47: Append `CoCCode` to each item in "Project" drop-down filter
Users would like to see the CoC Code associated with each project in the app (see image below for desired UX).  This will require capturing data from the `ProjectCoC.csv` file on upload (which currently is not happening).

image

### PR DESCRIPTION
**Pull request by IvanM26**
# Overview
Main development:
- CoC code was added next to project name choices in "Project" dropdown. Several changes were necessary to make this change (check Details for more information). These changes are meant to be incorporated once the database is reset.

In this PR I also included some minor changes:
- cleaned database-utils scripts (some code styling, file renaming and location changes)
- minor documentation update (there was an argument that wasn't used and was present in examples)

# Details
CoC code is present in ProjectCoC.csv, a file that is not currently processed in the app. I used Enrollment.csv processing as a reference, because the files have some characteristics in common (the need to use project_id instead of orig_project_id being the most noteworthy).

Required changes included:
- create read function to read .csv data (`read_project_coc()`)
- add SQL script that creates project_coc table in the database
- modify HMISmetadata to reflect that a new file is required
- include project_coc logic when creating dm object
- properly append coc code to project name
- make dropdown-options container be the body to solve a UI issue

# Test
- Run postgres/utils/recreate_dev.R
- Launch the application in dev and upload a test file
- Relaunch the application (to update the dev database)
- Check that coc code was appended accordingly

Closes #47

### CODE REVIEW

File: R/utils_ingest.R [hunk 1]
```diff
@@ -1088,6 +1077,62 @@ read_project <- function(file) {
...
+    # only read in columns needed for "PROJECT" database table
+    col_select = expected_colnames,
+    # define schema types
+    col_types = readr::cols(
+      .default = readr::col_character(),
+      DateUpdated = readr::col_date()
+    )
+  ) |>
+    janitor::clean_names(case = "snake") |>
+
```
Comment (mthomas-ketchbrook) [inline R/utils_ingest.R:1126]: Remove empty line

File: R/utils_ingest.R [hunk 2]
```diff
@@ -1088,6 +1077,62 @@ read_project <- function(file) {
...
+    "ZIP",
+    "GeographyType",
+    "DateUpdated"
+  )
+
+  check_colnames(file, expected_colnames)
+
+  readr::read_csv(
+    file = file,
+    # only read in columns needed for "PROJECT" database table
```
Comment (mthomas-ketchbrook) [inline R/utils_ingest.R:1117]: Change *"PROJECT"* to *"PROJECT CoC"*

**Review by mthomas-ketchbrook — APPROVED** [review]
Very minor formatting/comment changes requested

File: R/utils_ingest.R (hunk 1 above)
Comment (IvanM26) [reply R/utils_ingest.R:1126]: Done!

File: R/utils_ingest.R (hunk 2 above)
Comment (IvanM26) [reply R/utils_ingest.R:1117]: Done!

**Comment by trevinflick** [comment]
going through each PR that is going towards #128 ✅

**Pull request merged**